README.md

```diff
@@ -34,14 +34,24 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. If you were to describe semantic HTML to the next cohort of students, what would you say?
 
+Semantics give the bones of the website meaning, logic, and order. Gives the website a basic flow. 
+
 2. Name two big differences between ```display: block;``` and ```display: inline;```.
 
+Both are for displaying boxes on a webpage. Block displays boxes vertically. Inline displays content horizontally. Inline block displays boxes horizontally.
```

Comment:
:exclamation: `display: block;` and `display: inline;` do not necessarily relate to the direction of items in the page. The _effect_ may be what you described above, but that is because:

* Block elements take up the whole with of the container they're in. That means any elements would be forced to be under it, creating a column.
* Inline elements take up only the amount of width needed to fit its content. That means it is _possible_ to place other inline elements beside it to create a row.